02-electric/electric-section-three/electric-section-three.css

```diff
@@ -1,65 +1,327 @@
+  .electric-section-three-subcontainer{
+    grid-template-columns: repeat(2,1fr);
+    grid-template-areas:
+    "electric-section-three-title electric-section-three-title"
+    "electric-section-three-part-one-container electric-section-three-part-one-container"
+    "electric-section-three-part-two-container electric-section-three-part-three-container";
+    padding: var(--subcontainer-padding-desktop);
+    row-gap: var(--subsection-container-row-gap-desktop);
+    column-gap:var(--subsection-container-column-gap-desktop) ;
+  }
+}
+
+.electric-section-three-title {
+  grid-area:electric-section-three-title;
+  text-align: center;
+  font-size: var(--title-font-size-mobile);
+  font-weight: var(--title-font-weight-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-title {
+    font-size: var(--title-font-size-desktop);
+    font-weight: var(--title-font-weight-desktop);
+  }
+}
+
+/*****PART one******/
+
+.electric-section-three-part-one-container{
+  grid-area:electric-section-three-part-one-container;
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: var(--subcontainer-row-gap-mobile);
+  background-color: var(--level2-background-color);
+}
+
+@media (min-width:500px) {
+  .electric-section-three-part-one-container{
+    padding: var(--subcontainer-padding-desktop);
+  }
+}
+
+.electric-section-three-part-one-copy-container{
+  display: grid;
+  align-content: start;
+  row-gap: var(--copy-container-row-gap-mobile);
+  background-color: var(--level3-background-color);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-one-copy-container {
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
+}
+
+.electric-section-three-part-one-subtitle {
+  text-align: center;
+  font-size: var(--subtitle-font-size-mobile);
+  font-weight: var(--subtitle-font-weight-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-one-subtitle {
+    text-align: left;
+    font-size: var(--subtitle-font-size-desktop);
+    font-weight: var(--subtitle-font-weight-desktop);
+  }
+}
+
+.electric-section-three-part-one-paragraph-container {
+  display: grid;
+  row-gap: var(--copy-container-row-gap-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-one-paragraph-container{
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
+}
+
+.electric-section-three-part-one-paragraph {
+  text-align: center;
+  font-size: var(--paragraph-font-size-mobile);
+  font-weight: var(--paragraph-font-weight-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-one-paragraph {
+    text-align: left;
+    font-size: var(--paragraph-font-size-desktop);
+    font-weight: var(--paragraph-font-weight-desktop);
+  }
+}
+
+/*****PART TWO******/
+
+.electric-section-three-part-two-container{
+  grid-area:electric-section-three-part-two-container;
+  display: grid;
+  row-gap: var(--subcontainer-row-gap-mobile);
+  border: var(--subcontainer-border-mobile);
```

Comment:
looking at the figma
part two and part three do not have any borders on either mobile or desktop
but you added a border here ... is this deliberate?

Reply:
its a mistake . i removed